support/cas-server-support-oidc-core/src/main/java/org/apereo/cas/oidc/claims/OidcAddressScopeAttributeReleasePolicy.java

```diff
@@ -17,11 +15,11 @@
 public class OidcAddressScopeAttributeReleasePolicy extends BaseOidcScopeAttributeReleasePolicy {
     private static final long serialVersionUID = 1532960981124784595L;
 
-    private List<String> allowedAttributes = Stream.of("address").collect(Collectors.toList());
+    public static final List<String> ALLOWED_ATTRIBUTES = List.of("address");
```

Comment:
Since we are now making this public (and the ones below),

1. You should likely add a bit of Javadoc here. (Checkstyle is probably mad at this line already)
2. It would make better sense to me to rename these as ALLOWED_CLAIMS to stick with the accepted jargon though there is the getter below that cannot be changed. So, your call.